components/icons/Icons.tsx

```diff
@@ -0,0 +1,84 @@
+import { FC } from 'react';
+import { tw } from '../../utils';
+
+interface IconProps {
+  onClick?: () => void;
+  className?: string;
+}
+
+export const ArrowIcon: FC<IconProps> = ({ onClick, className }) => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      fill='none'
+      viewBox='0 0 24 24'
+      stroke-width='1.5'
+      stroke='currentColor'
+      className={tw('w-6 h-6 cursor-pointer', className)}
+      onClick={onClick}
```

Comment:
I just noticed this. Why is an svg element clickable? is it always clickable? Normally you should have this inside of some kind of <button>